### ISSUE TITLE
[Bug?]: The editor could not be opened due to an unexpected error: Unable to read file

### ISSUE THREAD

**Issue by wegry** [yarnpkg/berry#4550]
### Self-service

- [ ] I'd be willing to implement a fix

### Describe the bug

Trying to go to declarations with the typescript@4.7 triggers this message with yarn@3.2.1 and arcanis.vscode-zipfs@3.0.0.

![Screen Shot 2022-06-15 at 07 43 04](https://user-images.githubusercontent.com/5156873/173819514-47a1c470-05fb-4edd-b012-fcbdc8959412.png)

### To reproduce

packageJsonAndInstall({
  dependencies: {
    "react-menu-list": "6.1.1",
    "typescript": "4.7.2"
  }
})

VS Code 1.68.0
with yarn@3.2.1 and PnP enabled.

```typescript
import {MenuList} from 'react-menu-list`
```

And try to go to Type Definition within VS Code.

### Environment

```shell
System:
    OS: macOS 12.4
    CPU: (8) arm64 Apple M1 Pro
  Binaries:
    Node: 16.15.1 - /private/var/folders/k5/x6ks_wdd1b31y221zccrmbtm0000gn/T/xfs-7d58a0df/node
    Yarn: 3.2.1 - /private/var/folders/k5/x6ks_wdd1b31y221zccrmbtm0000gn/T/xfs-7d58a0df/yarn
    npm: 8.11.0 - ~/.nvm/versions/node/v16.15.1/bin/npm
```

**Comment by wegry** [comment]
Looks like running `yarn dlx @yarnpkg/sdks vscode` patched this up for me.

**Comment by merceyz** [comment]
Indeed, this was fixed in https://github.com/yarnpkg/berry/pull/4485
